FAQ.md

```diff
@@ -0,0 +1,19 @@
+Q: When running pytest for a code challenge, I'm getting unexpected results or errors. What should I check?
+
+A: Before running pytest, it's essential to check your current directory (path) to ensure that you are in the correct location for the specific code challenge. Running pytest from an incorrect path may cause errors or unexpected results because the test files or modules may not be found. To verify your location, you can use the pwd command (on macOS/Linux) or cd and dir commands (on Windows) to navigate to the correct directory. Once confirmed, you can run pytest to test your code accurately.
+
+Q: When running pytest, I noticed that after specifying the filename, the rest of the command (like -p no:cacheprovider) isn’t necessary. Can I simplify the command? 
+
+A: Yes! When running pytest on a specific test file, you usually only need to provide the path to the file. For example, to test all_construct_test.py, you can simply run:
+
+pytest all-construct/all_construct_test.py
+
+The extra option -p no:cacheprovider is typically used to disable caching but is not required unless you have a specific reason to avoid pytest’s caching. In most cases, just specifying the filename is sufficient to run your tests.
+
+Q: Do I need to add -p no:cacheprovider after the filename when running pytest?
+
+A: No, you don’t need it. You can simply run:
+
+pytest all-construct/all_construct_test.py
+The -p no:cacheprovider option is only needed in special cases to disable caching, so it’s usually fine to leave it out.
```

Comment:
please note that we will need to ignore the cache if you have updated a file. If users are not seeing their code changes reflected in the test results, adding this argument will ensure that the interpreter is using the most recent code.